css(fix): mention forward animations keep new staking context (#35056)
mdn/content: files/en-us/web/css/css_positioned_layout/understanding_z-index/stacking_context/index.md

- Element with a {{cssxref("container-type")}} value `size` or `inline-size` set, intended for [container queries](/en-US/docs/Web/CSS/CSS_containment/Container_queries).
- Element that is a child of a [flex](/en-US/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox) container, with {{cssxref("z-index")}} value other than `auto`.
- Element that is a child of a {{cssxref("grid")}} container, with {{cssxref("z-index")}} value other than `auto`.

> - Element with an {{cssxref("opacity")}} value less than `1`.

Reviewer: ```suggestion
- Element with an {{cssxref("opacity")}} value less than or equal to `1`.
```

Reviewer: I think? This seems to be what the original issue suggests.

Author: The `equal to 1` case is an exception for only the `@keframes`. Which we've covered in the last bullet point on line 44.

Reviewer: I see what you mean, yup. That's fine.

- Element with a {{cssxref("will-change")}} value specifying any property that would create a stacking context on non-initial value (see [this post](https://dev.opera.com/articles/css-will-change-property/)).
- Element with a {{cssxref("contain")}} value of `layout`, or `paint`, or a composite value that includes either of them (i.e. `contain: strict`, `contain: content`).
- Element placed into the [top layer](/en-US/docs/Glossary/Top_layer) and its corresponding {{cssxref("::backdrop")}}. Examples include [fullscreen](/en-US/docs/Web/API/Fullscreen_API) and [popover](/en-US/docs/Web/API/Popover_API) elements.

> - Element that got animated, with [`animation-fill-mode`](/en-US/docs/Web/CSS/animation-fill-mode) set to [`forwards`](/en-US/docs/Web/CSS/animation-fill-mode#forwards), where staking context creating properties were animated.

Reviewer: ```suggestion
- Element that has had stacking context-creating properties (such as `opacity`) animated using {{cssxref("@keyframes")}}, with [`animation-fill-mode`](/en-US/docs/Web/CSS/animation-fill-mode) set to [`forwards`](/en-US/docs/Web/CSS/animation-fill-mode#forwards).
```

Reviewer: It is just keyframe animations we're talking about, right? Not transitions?
